packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -122,6 +124,9 @@ export class RelayerConfig {
 
     assert(this.crossDomainFinalizationThreshold < 100, "CROSS_DOMAIN_FINALIZATION_THRESHOLD must be < 100");
 
+    this.relayerDiscount = RELAYER_DISCOUNT ? Number(RELAYER_DISCOUNT) : 0;
+    assert(this.relayerDiscount >= 0 && this.relayerDiscount <= 100, "RELAYER_DISCOUNT must be between 0 and 100");
```

Comment:
Can you merge master in and replace this assert with a throw?

Reply:
done.